src/components/CreateAddress/PublicKeyImporter.jsx

```diff
@@ -332,6 +346,16 @@ class PublicKeyImporter extends React.Component {
             publicKeyImporter.conflict && (
               <Conflict message="Warning, BIP32 path is in conflict with the network and address type settings.  Do not proceed unless you are absolutely sure you know what you are doing!" />
             )}
+          {validatePublicKey(publicKeyImporter.publicKey, addressType).includes(
```

Comment:
I don't think we need to do this twice (running it in the message too). Maybe run the validation before return and if it has length then pass that to `Conflict`

Reply:
fair, dont need to run this twice

but there is the case when the error is `Public key cannot be blank.` which has length... so need to use a slightly different criteria there..

Reply:
Awesome. Good call and works for me.